### ISSUE TITLE
grpc - temporary protobuf descriptors are not cleaned up

### ISSUE THREAD

**Issue by twilly** [uber/prototool#95]
`prototool grpc` leaves behind temporary protobuf descriptors on every call.

To reproduce, make any call with `grpc` and inspect your OS's typical temp file location. For example:

```
$ prototool grpc foo.Bar/Baz ''
$ ls /tmp/prototool*
/tmp/prototool661509915  /tmp/prototool892888492
```

**Comment by peter-edge** [comment]
Closed by #96

**Comment by peter-edge** [comment]
This is fixed on v0.3.0.
